src/ray_julia_jll/ray_julia_jll.jl

```diff
@@ -11,13 +11,12 @@ using libcxxwrap_julia_jll
 
 abstract type BaseID end
 
-@wrapmodule(joinpath(artifact"ray_julia", "julia_core_worker_lib.so"))
+@wrapmodule(() -> joinpath(artifact"ray_julia", "julia_core_worker_lib.so"))
```

Comment:
what's this adding?

Reply:
This is required by newer versions of CxxWrap:
```
┌ Error: Calling `@wrapmodule` with the path of the library to load is no longer supported.
│ Pass the name of a function returning the path instead, e.g. use `libfoo_jll.get_libfoo_path` instead of `libfoo_jll.libfoo`.
└ @ CxxWrap.CxxWrapCore ~/.julia/packages/CxxWrap/66SRr/src/CxxWrap.jl:729
```
I didn't look into this but it looks to me like its for better support for BinaryBuilder libraries